src/requestor/routes/manage.py

```diff
@@ -36,6 +36,39 @@ class CreateRequestInput(BaseModel):
     policy_id: str = None
 
 
+async def grant_or_revoke_arborist_policy(
+    arborist_client, policy_id, username, revoke, status
```

Comment:
you don't need `status` anymore in this function, unless you want to add it to the log on line 45